### PR TITLE
fix week view

### PR DESCRIPTION
**Pull request by szaimen**
## Before
https://user-images.githubusercontent.com/42591237/127358658-68bfe213-1c6b-4845-9450-5e3744f95b73.mp4

## After
https://user-images.githubusercontent.com/42591237/127358455-ea98b2b9-1072-41a5-af27-5e0b74f11248.mp4

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

**Review by st3iny — APPROVED** [review]
Works well. Just some ultra minor code styling issues.

File: css/fullcalendar.scss [hunk 1]
```diff
@@ -193,6 +193,17 @@
 	}
 }
 
+// Fix week view
+.fc-col-header-cell {
+	word-break: break-word;
+	white-space: normal;
+}
```
[the remark this answers is not in the document]
Comment (st3iny) [reply css/fullcalendar.scss:200]: nvm

**Review by GretaD — APPROVED** [review]

**Comment by ChristophWurst** [comment]
https://github.com/nextcloud/calendar/pull/3382 it's already there :laughing:

**Pull request merged**